File: VB/WebLayoutSolution.Module/BusinessObjects/BO.vb
Imports DevExpress.ExpressApp.Model
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo

Namespace WebLayoutSolution.Module.BusinessObjects

    <NavigationItem>
    Public Class BusinessObject
        Inherits BaseObject

        Public Sub New(ByVal s As Session)
            MyBase.New(s)
        End Sub

        <ModelDefault("Caption", "Abcd")>
        Public Property FieldA As String

        <ModelDefault("Caption", "Efghijklmn")>
        Public Property FieldB As String

        Public Property FieldC As String

        Public Property FieldD As String

        Public Property FieldE As String

        <ModelDefault("Caption", "A long caption")>
        Public Property FieldF As String

        Public Property FieldG As String

        Public Property FieldH As String

        <ModelDefault("Caption", "Another long caption")>
        Public Property FieldI As String

        Public Property FieldJ As String

        Public Property FieldK As String

        Public Property FieldL As String

        Public Property FieldM As String

        <ModelDefault("Caption", "A very very very very very long caption")>
        Public Property FieldN As String
    End Class
End Namespace

File: CS/WebLayoutSolution.Module/BusinessObjects/BO.cs
using DevExpress.ExpressApp.Model;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Xpo;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WebLayoutSolution.Module.BusinessObjects {
    [NavigationItem]
    public class BusinessObject : BaseObject {
        public BusinessObject(Session s) : base(s) { }
        [ModelDefault("Caption", "Abcd")]
        public string FieldA { get; set; }
        [ModelDefault("Caption", "Efghijklmn")]
        public string FieldB { get; set; }
        public string FieldC { get; set; }
        public string FieldD { get; set; }
        public string FieldE { get; set; }
        [ModelDefault("Caption", "A long caption")]
        public string FieldF { get; set; }
        public string FieldG { get; set; }
        public string FieldH { get; set; }
        [ModelDefault("Caption", "Another long caption")]
        public string FieldI { get; set; }
        public string FieldJ { get; set; }
        public string FieldK { get; set; }
        public string FieldL { get; set; }
        public string FieldM { get; set; }
        [ModelDefault("Caption", "A very very very very very long caption")]
        public string FieldN { get; set; }
    }
}
